frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -423,10 +424,13 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
             },
         ],
     })),
-    reducers(({ props }) => ({
+    reducers(({ props, values }) => ({
         orderBy: [
```

Comment:
Want thing that was not obvious to me: can I delay assignment until the feature is actually used?  Or is that what the "release conditions" are for in the feature UI?